source/authentication/email-password.txt

```diff
@@ -20,6 +20,20 @@ Overview
 Email/password authentication lets users register and login using an email address.
 Atlas App Services must confirm Email/Password users before they may log in.
 
+Users may choose one of three ways to handle user confirmation:
```

Comment:
Right above this, we're talking about "Users" in the context of end users of an app registering and logging in, but here I think you're talking about developers, so I would avoid "Users" here.

```suggestion
Developers may choose one of three ways to handle user confirmation:
```

Reply:
After reading the page, I think having this "three ways to handle user confirmation" section in the Overview is a little strange. I would expect to find this down below in the "User Confirmation" section. (And I see that a summary _is_ there as I would expect.) I think of the Overview as more of a high-level description and the three ways of handling user confirmation seem like implementation details that have been sort of shoe-horned in here.